src/pages/ReactionPage/ReactionPage.tsx

```diff
@@ -21,7 +21,9 @@ const ReactionPage = () => {
       )
     ) ?? [];
 
-  const hasPartygoers = useMemo(() => partygoers.length > 0, [partygoers]);
+  const hasPartygoers = useMemo(() => recentWorldUsers.length > 0, [
+    recentWorldUsers,
+  ]);
```

Comment:
This doesn't need to be `useMemo`'d

```suggestion
  const hasPartygoers = recentWorldUsers.length > 0;
```